.github/workflows/test_linux_jax_wheels.yml

```
@@ -0,0 +1,183 @@
+name: Test JAX Wheels
+on:
+  workflow_call:
+    inputs:
+      amdgpu_family:
+        description: GPU family to test
+        required: true
+        type: string
+      release_type:
+        description: The type of release ("nightly" or "dev")
+        required: true
+        type: string
+      s3_subdir:
+        description: S3 subdirectory (e.g., v2 or v2-staging)
```

```suggestion
        description: S3 subdirectory
```
Outdated for prereleases and nightlies to follow.